Source/ACE.Server/Entity/Landblock.cs

```diff
@@ -766,10 +766,13 @@ private bool AddWorldObjectInternal(WorldObject wo)
                     wo.CurrentLandblock = null;
 
                     if (wo.Generator != null)
-                        log.Debug($"AddWorldObjectInternal: couldn't spawn 0x{wo.Guid}:{wo.Name} from generator {wo.Generator.WeenieClassId} - 0x{wo.Generator.Guid}:{wo.Generator.Name}");
+                    {
+                        log.Debug($"AddWorldObjectInternal: couldn't spawn 0x{wo.Guid}:{wo.Name} [{wo.WeenieClassId} - {wo.WeenieType}] from generator {wo.Generator.WeenieClassId} - 0x{wo.Generator.Guid}:{wo.Generator.Name}");
+                        wo.NotifyOfEvent(RegenerationType.PickUp); // Notify generator the generated object is effectively destroyed, use Pickup to catch both cases.
```

Comment:
is it possible this could send some generators into an infinite loop?

Reply:
it would keep re-trying spawn per regen rate + delay rate rules

Reply:
depending on generator it also could select another profile